packages/renderer/src/lib/deployments/deployment-utils.ts

```diff
@@ -22,23 +22,48 @@ import type { DeploymentUI } from './DeploymentUI';
 import type { V1Deployment } from '@kubernetes/client-node';
 
 export class DeploymentUtils {
-  humanizeAge(started: string): string {
+  humanizeAge(started: string | undefined): string {
+    if (!started) {
+      return '';
+    }
     // get start time in ms
     const uptimeInMs = moment().diff(started);
     // make it human friendly
     return humanizeDuration(uptimeInMs, { round: true, largest: 1 });
   }
 
+  refreshAge(deployment: DeploymentUI): string {
+    if (!deployment.created) {
+      return '';
+    }
+    // make it human friendly
+    return this.humanizeAge(deployment.created.toString());
```

Comment:
I'm wondering why we use the toString and do not send the `.created` field directly

humanizeAge should be smarter

Reply:
Yes, this was poor. Updated humanizeAge to accept Date.